src/static/js/modules/explore-rates.js

```diff
@@ -1195,6 +1207,11 @@ $( '.calc-loan-amt, .credit-score' ).on( 'keyup', '.recalc', function( evt ) {
   }
 } );
 
+$( '.calc-loan-amt, .credit-score' ).on( 'blur', '.recalc', function( evt ) {
```

Comment:
`$('.calc-loan-amt .recalc')` includes the `#county` select, so `addCommas` is being called when it blurs and changing its value.